yaksh/live_server_tests/selenium_test.py

```diff
@@ -27,6 +27,8 @@ class SeleniumTestError(Exception):
 class SeleniumTest():
     def __init__(self, url, quiz_name, module_name, course_name):
         self.driver = webdriver.Firefox()
+        self.driver.set_window_position(0, 0)
+        self.driver.set_window_size(1024, 768)
```

Comment:
Just out of curiousity, could you tell me what was the new window size was before rescaling the size to 1024*768?

Reply:
The selenium browser that is run by Travis CI seems to run in a smaller window size, this coupled with the fact that we have responsive templates leads to Selenium failing because it could not find certain elements (which are there but are hidden due to scroll/Accordion buttons and other responsive features of Bootstrap).

Explicitly declaring the screen size ensures this does not happen.